saleor/settings.py

```diff
@@ -28,13 +28,12 @@ def get_list(text):
 
 INTERNAL_IPS = get_list(os.environ.get('INTERNAL_IPS', '127.0.0.1'))
 
+# Some cloud providers like Heroku export REDIS_URL variable instead of CACHE_URL
+REDIS_URL = os.environ.get('REDIS_URL')
+if REDIS_URL:
+    CACHE_URL = os.environ.setdefault('CACHE_URL', REDIS_URL)
```

Comment:
Question: won't this actually set the env variable? I'm asking because the env memory is limited and there are situations where setting a new variable could silently unset another.

Reply:
No it won't. I've checked it. 
I've ran python interprete as follows:
```bash
$ REDIS_URL="redis://localhost:6379/1" CACHE_URL="locmem://" ipython
```

```
In [1]: import os

In [2]: CACHE_URL=os.environ.get('CACHE_URL')

In [3]: REDIS_URL = os.environ.get('REDIS_URL')

In [4]: CACHE_URL = os.environ.setdefault('CACHE_URL', REDIS_URL)

In [5]: print(CACHE_URL)
locmem://

In [6]: print(REDIS_URL)
redis://localhost:6379/1
```

So if both variables are set - `CACHE_URL` will remain as it was set in environment. Otherwise it will inherit value from `REDIS_URL`